### ISSUE TITLE
extra empty rdt_TableHeader above table using up space

### ISSUE THREAD

**Issue by mdodge-ecgrow** [jbetancur/react-data-table-component#756]
## Issue Check list
- [x ] Agree to the [Code of Conduct](https://github.com/jbetancur/react-data-table-component/blob/master/CODE-OF-CONDUCT.md)
- [x ] Read the README
- [x ] You are using React 16.8.0+
- [x ] You installed `styled-components`
- [x ] Include relevant code or preferably a [code sandbox](https://codesandbox.io/embed/react-data-table-sandbox-ccyuu
)

## Describe the bug
For some reason there is an extra div being generated above my table with class `rdt_TableHeader`.  This div is empty but using up lots of white space.

## Expected behavior
This extra div not to be there.

## Code Sandbox, Screenshots, or Relevant Code
Please include a codesandbox to help **expedite** troublshooting.

Otherwise, add screenshots and/or complete sample code to help explain your problem.
```js
const customStyles = {
		rows: {
			style: {
				// minHeight: '72px' // override the row height
			}
		},
		headCells: {
			style: {
				// paddingLeft: '8px', // override the cell padding for head cells
				// paddingRight: '8px',
				fontWeight: 'bold',
				fontSize: '15px'
			}
		},
		cells: {
			style: {
				// paddingLeft: '8px', // override the cell padding for data cells
				// paddingRight: '8px'
			}
		}
	};

	const ProductsGrid = () => {
		return (
			<DataTable
				data={gridRows}
				columns={columns}
				customStyles={customStyles}
			/>
		);
	};

	return (
	...
	<ProductsGrid />
	...
	)
```

## Versions (please complete the following information)
 - React (RDT requires 16.8.0+) 17.0.1
 - Styled Components 5.2.1
 - OS: [e.g. iOS] Win10 x64
 - Browser [e.g. chrome, safari, firefox] Firefox

## Additional context
Here is the extra div:
![image](https://user-images.githubusercontent.com/57459547/104763041-f3797b80-572a-11eb-8b17-afcf64ef4da1.png)

**Comment by mdodge-ecgrow** [comment]
And I can't do `display: none` on the `rdt_TableHeader` class because it is getting overridden by the auto-generated random classes.

**Comment by jbetancur** [comment]
That’s a good point. Currently without noHeader the header shows always even if you omit title or actions props . It should probably be if no title or actions props always hide the header. And noHeader can remain as a way to programmatically hide

**Comment by jbetancur** [comment]
I’m going to flag this as a feature for the next version of rdt version 7.x

**Comment by jbetancur** [comment]
Available in 7.0.0-alpha-5

**Comment by rafeldev** [comment]
> noHeader

this really worked for me
